util/env_windows_test.cc

```diff
@@ -29,7 +84,7 @@ TEST_F(EnvWindowsTest, TestOpenOnRead) {
   ASSERT_LEVELDB_OK(env_->GetTestDirectory(&test_dir));
   std::string test_file = test_dir + "/open_on_read.txt";
 
-  FILE* f = std::fopen(test_file.c_str(), "w");
+  FILE* f = std::fopen(test_file.c_str(), "wN");
```

Comment:
N is a non-standard extension that most developers will not be familiar with. Can you add a comment explaining it? Here is some draft text.

N is a Windows-specific extension that makes the file handle non-inheritable by child processes. Without this option, open handles in child processes may prevent the database from getting deleted.

Reply:
Added in WindowsEnv::NewLogger. Should I add this here too?